### PR TITLE
Improve S3415 ('inverted-assertion-arguments'): Report consistent error message

### LINKED ISSUE #3078: Improve S3415 ('inverted-assertion-arguments'): Report consistent location and error message

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**
Fixes #3078

### CODE REVIEW

File: eslint-bridge/src/linting/eslint/rules/inverted-assertion-arguments.ts [hunk 1]
```diff
@@ -69,10 +69,10 @@ export const rule: Rule.RuleModule = {
 function checkInvertedArguments(node: estree.CallExpression, context: Rule.RuleContext) {
   const args = extractAssertionsArguments(node);
   if (args) {
-    const [actual, expected] = args;
+    const [actual, expected, format] = args;
```
Comment (ilia-kebets-sonarsource) [inline eslint-bridge/src/linting/eslint/rules/inverted-assertion-arguments.ts:72]: rename to `correctFormat`?

**Comment by ilia-kebets-sonarsource** [comment]
So there was no implementation changes on the location, just added testing for it?
If so, can you rename the PR to express it?
Otherwise, LGTM

File: eslint-bridge/src/linting/eslint/rules/inverted-assertion-arguments.ts (hunk 1 above)
Comment (yassin-kammoun-sonarsource) [reply eslint-bridge/src/linting/eslint/rules/inverted-assertion-arguments.ts:72]: `correctFormat`? Is there a wrong format?

**Comment by yassin-kammoun-sonarsource** [comment]
> So there was no implementation changes on the location, just added testing for it?
> If so, can you rename the PR to express it?
> Otherwise, LGTM

The ticket's title wanted to express inconsistency between the error message and the location of the actual/expected parameters.

File: eslint-bridge/src/linting/eslint/rules/inverted-assertion-arguments.ts (hunk 1 above)
Comment (ilia-kebets-sonarsource) [reply eslint-bridge/src/linting/eslint/rules/inverted-assertion-arguments.ts:72]: Well, we have found a wrong format, and displaying what the right one should be.

**Comment by ilia-kebets-sonarsource** [comment]
> > So there was no implementation changes on the location, just added testing for it?
> > If so, can you rename the PR to express it?
> > Otherwise, LGTM
> 
> The ticket's title wanted to express inconsistency between the error message and the location of the actual/expected parameters.

Ah, I though of location as start/end (line,col) pairs where the issue gets flagged. This was not tested before this PR.

**Pull request merged**